public/js/message_box.js

```diff
@@ -0,0 +1,51 @@
+/*global myRadioAPIKey:false*/
+/*eslint no-undef: "error"*/
+/*eslint no-console: ["error", { allow: ["warn", "error"] }] */
```

Comment:
i'm being picky at this point, but I wouldn't recommend disabling these warnings for the whole file. Just the `/*global ...*/` definition will work for the no-undef, and an inline `// eslint-disable-line no-console` will work for the console statement